test/unit/notify/send-emails.test.js

```diff
@@ -28,25 +28,58 @@ describe('send emails', () => {
     defaultEmailAddresses = env.notifyEmailAddresses.split(',')
 
     jest.mock('uuid', () => ({ v4: () => mockReference }))
+
+    formatEmailAddresses.mockReturnValue(
+      emailAddresses
+    )
   })
 
   afterEach(() => {
     jest.clearAllMocks()
```

Comment:
Just seeing in this file (done by me!) there are `jest.clearAllMocks()` and `jest.clearAllMocks()` in the `beforeEach` as well as the `afterEach` -- can we move these into `afterEach`?